src/flaskr/templates/main/home.html

```diff
@@ -16,29 +16,44 @@ <h1>{% block title %}Home Page | Operations Optimization{% endblock %}</h1>
     <label for="time_increment">Time Increment</label>
     <input name="time_increment" id="time_increment" required>
+
+    <label for="cleanse">Cleanse</label>
+    <input type="checkbox" name="cleanse" id="cleanse">
+    <label for="predict">Predict</label>
+    <input type="checkbox" name="predict" id="predict">
+    <label for="prediction_date">Prediction End Date</label>
+    <input type="text" name="prediction_date" id="prediction_date" placeholder="YYYY-MM-DD HH:MM:SS" pattern="\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}" required title="Date format should be YYYY-MM-DD HH:MM:SS">
+
+    <label for="line">Line Chart</label>
+    <input type="radio" name="chartType" id="line" value="line" required>
+    <label for="bar">Bar Chart</label>
+    <input type="radio" name="chartType" id="bar" value="bar" required>
+    <label for="radar">Radar Chart</label>
+    <input type="radio" name="chartType" id="radar" value="radar" required>
+
     <button type="submit">Get Data</button>
+
   </form>
 
-  <canvas id="lineChart" width="800" height="400"></canvas> <!-- Canvas element where the line chart will be drawn -->
+  <canvas id="Chart" width="800" height="400"></canvas> <!-- Canvas element where the line chart will be drawn -->
   <script>
     // Get the context of the canvas element we just defined
-    var ctx = document.getElementById('lineChart').getContext('2d');
+    var ctx = document.getElementById('Chart').getContext('2d');
 
-    // Creating a new Chart instance and passing the context
-    var lineChart = new Chart(ctx, {
-      type: 'line', // Specifies that we want to create a line chart
+    var chart = new Chart(ctx, {
+      type: '{{chart_type}}', // Specifies that we want to create a scatter chart
       data: {
         labels: {{labels | safe}}, // Sets the labels for the x-axis from a template variable
-        datasets: [{ // Data for the line chart including the dataset
+        datasets: [{ // Data for the bar chart including the dataset
           label: 'Sensor', // Label for the dataset
-          data: {{values | safe}}, // The actual data points for the line chart from a template variable
-          borderColor: 'rgba(255, 99, 132, 1)', // Color of the line
-          fill: false, // Specifies that the area under the line should not be filled
-          borderWidth: 1 // Width of the line
+          data: {{values | safe}}, // The actual data points for the bar chart from a template variable
+          backgroundColor: 'rgba(255, 99, 132, 1)', // Color of the bars
+          fill: false,
+          borderWidth: 1 // Width of the bars
         }]
       },
       options: {
-        responsive: false // Chart will not resize automatically when the container does
+        responsive: true // Chart will not resize automatically when the container does
```

Comment:
i may be wrong, but if its responsive it will ignore the canvas parameters and size itself however it needs on the page to fit nicely

Reply:
Yes, thats what I wanted to happen, it looks better than if only fills up 5/8 of the screen